sinks/kafka/kafka.go

```diff
@@ -23,6 +23,9 @@ import (
 var _ sinks.MetricSink = &KafkaMetricSink{}
 var _ sinks.SpanSink = &KafkaSpanSink{}
 
+const totalMetricsFlushedMetricKey = "sink.metrics_flushed_total"
+const totalSpansFlushedMetricKey = "sink.spans_flushed_total"
```

Comment:
Can you put these constants (and the others in the other packages) under an exported name in the `sinks` package? That'd cut down on a ton of duplication (and potential for typos!) & allow us to document there that sinks are expected to emit those.